components/record/MetadataField.vue

```diff
@@ -56,5 +90,14 @@
   ul {
     list-style: none;
     padding: 0;
+    li {
+      display: inline;
+      &:after {
```

Comment:
You can have it even like this: `&:not(:last-child):after` and then remove the `&:last-child:after`